internal/objects/service/service.go

```diff
@@ -49,10 +50,23 @@ const (
 	// TODO [danehans]: Make proxy protocol configurable or automatically enabled. See
 	// https://github.com/projectcontour/contour-operator/issues/49 for details.
 	awsLbBackendProtoAnnotation = "service.beta.kubernetes.io/aws-load-balancer-backend-protocol"
+	// awsLbEipAnnotation is a Service annotation that provides capability to assign Load
+	// Balancer IP based on AWS Elastic IP resource in external LoadBalancer scope
+	awsLbEipAnnotation = "service.beta.kubernetes.io/aws-load-balancer-eip-allocations"
 	// awsProviderType is the name of the Amazon Web Services provider.
 	awsProviderType = "AWS"
+	// azureLbSubnetAnnotation is a Service annotation that provides capability to assign
+	// Load Balancer IP based on desired subnet in internal LoadBalancer scope
+	azureLbSubnetAnnotation = "service.beta.kubernetes.io/azure-load-balancer-internal-subnet"
+	// azureLbResourceGroupAnnotation is a Service annotation that provides capability to
+	// assign Load Balancer IP based on Public IP Azure resource that resides in different
+	// resource group as Azure Kubernetes Service cluster in external LoadBalancer scope
```

Comment:
s/in external LoadBalancer scope/when load balancer scope is set to "External"./. Repeat for other providers.

Reply:
Resolved